common/vueapp/Notifiers.vue

```diff
@@ -238,17 +243,40 @@
             @selected-roles-updated="updateNotifierRoles"
             :display-text="notifier.roles && notifier.roles.length ? undefined : 'Share with roles'"
           /> <!-- /notifier sharing -->
+          <template v-if="parentApp === 'parliament' && notifier.alerts">
```

Comment:
should we still show this in viewer in case you edit a parliament notifier in viewer?

Reply:
My thought is that you can use notifiers for parliament or viewer. But when you're in parliament you want to configure which types of alerts you get from your parliament. And when you're in viewer you just want to add them to crons/hunts. I think it would be confusing to have the alerts show up for viewer since they don't pertain to viewer.